rero_ils/modules/documents/templates/rero_ils/_documents_get.html

```diff
@@ -66,14 +66,14 @@
   {% endif %}
 
   <!-- Holdings -->
-  {% if holdings %}
+  {%- if holdings %}
```

Comment:
Why we have to left strip holdings here, are the holdings correct?

Reply:
I use the lstrip to limit unrelevant spaces in generated HTML.
Holdings are still display as well as possible (test with every known holdings type)